tests/scripts/setup

```diff
@@ -46,16 +49,19 @@ def run_puppet_on_nodes
   #This initial agent run handles pxp-service init in the
   #puppet_enterprise::profile::agent profile class and pluginsync. Doing this
   #in setup makes the first user initiated run happen faster and more clearly
-  #show the changes the user has introduced.
+  #show the changes the user has introduced. It also ensures that the pxp-agent
+  #service is configured and running for quests that rely on puppet job run
+  #commands.
 
   manifests_dir = '/etc/puppetlabs/code/environments/production/manifests/'
 
   begin
 
     #Using the --tags puppet_enterprise::profile::agent argument ensures that
-    #only the resources in the PE agent profile are run, any compilation errors
-    #still abort the run. For clean catalog compilation, substitute empty site.pp
-    #file, then revert to the original after the runs ar complete.
+    #only the resources in the PE agent profile are run, but compilation errors
```

Comment:
Small nit - uppercase "only"

Reply:
@greglarkin This one is correct as is. It's part of a sentence that begins before the deleted lines.

Reply:
Woops, sorry ;)